### DISCUSSION TITLE
如何部署到leancloud的云引擎？

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#1035] [Q&A] [answered]
突然间发现 leancloud 平台中也有个云引擎部署，不知道如何将　ｗaline 服务部署到这个环境上面呢？

**Comment by lizheming** [comment, marked as answer]
应该基于独立运行的方案稍微适配下就可以了，你可以自己尝试下 https://waline.js.org/guide/server/vps-deploy.html#%E7%9B%B4%E6%8E%A5%E8%BF%90%E8%A1%8C-%E6%8E%A8%E8%8D%90

**Reply by elkan1788** [reply]
不熟悉前端开发，估计得找个时间学习琢磨下吧。

**Reply by lizheming** [reply]
哦，你是写 Java 的是吗？

**Reply by elkan1788** [reply]
是的，平时主要是 JAVA & PYTHON，前端只是l略懂些 HTML & JS & CSS。
